components/permissions/contexts/brave_wallet_permission_context.cc

```diff
@@ -334,4 +334,41 @@ bool BraveWalletPermissionContext::ResetPermission(
   return true;
 }
 
+// static
+std::vector<std::string>
+BraveWalletPermissionContext::GetWebSitesWithPermission(
+    ContentSettingsType content_settings_type,
+    content::BrowserContext* context) {
+  HostContentSettingsMap* map =
+      PermissionsClient::Get()->GetSettingsMap(context);
+  ContentSettingsForOneType settings;
+  map->GetSettingsForOneType(content_settings_type, &settings);
+
+  std::vector<std::string> result;
+  for (const auto& setting : settings) {
+    if (setting.GetContentSetting() != CONTENT_SETTING_ALLOW) {
+      continue;
+    }
+    result.push_back(setting.primary_pattern.ToString());
```

Comment:
nit: can be simplified as
```
if (setting.GetContentSetting() == CONTENT_SETTING_ALLOW) {
  result.push_back(setting.primary_pattern.ToString());
}
```

Reply:
yeah, I also noticed that, after CI has already started. Not a big deal imho, I just don't want to run it one more time.

Reply:
Okay, I'm fine with merging as is.